lib/Operations/Structured.coffee

```diff
@@ -155,6 +155,19 @@ module.exports = ()->
     getFirstOperation: ()->
       @beginning.next_cl
 
+    # get the next non-deleted operation
+    getNext: (start)->
```

Comment:
`start.getNext()` yields the same result. You can also call `start.getNext(i)` to get the i-th non-deleted operation